docs/sources/panels-visualizations/visualizations/trend/index.md

```diff
@@ -0,0 +1,31 @@
+---
+keywords:
+  - grafana
+  - graph panel
+  - trend panel
+  - documentation
+  - guide
+  - graph
+title: Trend
+weight: 90
+---
+
+# Trend
+
+The trend visualization type is the default and primary way to visualize data as a graph where x != time. It can render series as lines, points, or bars. It is versatile enough to display almost any sequential data.
+
+The trend panel supports all of the [visual options]({{< relref "../time-series" >}}) available to the time series panel with the exception of:
```

Comment:
We call it the visualization type in the previous section, but the panel here. Do these mean different things?